### DISCUSSION TITLE
Using currents with a --cloud-config-file in Typescript

### DISCUSSION THREAD

**Discussion by kateengstrom** [currents-dev/cypress-currents-reporter#170] [Q&A] [answered]
I'm switching from cy2 to cypress-cloud and using the debugger have found out why my config file is not being loaded successfully. Is there something I can do to make it read a ts config file?

`currents:config failed loading config file from: TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts" for /Users/kate/Repos/private/cypress/config/cypress.test.config.ts `

```
 WARNING  Failed to load config file, falling back to the default config. Attempted locations: [
  'file:///Users/kate/Repos/private/cypress/config/cypress.test.config.ts'
]
```

**Comment by agoldis** [comment]
This one should help: https://github.com/currents-dev/cypress-cloud#configuration-file-discovery

**Reply by kateengstrom** [reply]
Hi there, thank you. Unfortunately, this does not mention the TS-specific issue.

**Comment by agoldis** [comment, marked as answer]
@kateengstrom it simply doesn't support `ts` files - only vanilla JS

**Reply by kateengstrom** [reply]
Ah, thanks. Wish we didn't have to switch.
